index.md

```diff
@@ -5,4 +5,19 @@ Albeit tiny, this is our attempt to give back to the eco system in some way.
 
 Please fill up the [questionaire](https://goo.gl/forms/l6XTY2CISpEx2Vz02) so that we get a better hang of things you might expect of us.
 
-It will be a 2 hour session and a working knowledge of JavaScript is a must. Please read up [https://developer.mozilla.org/en-US/docs/Web/JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript)
+
+
+
+The bootcamp will be of two parts of about 3-4 hours each. In first part of the camp we will learn and build a working application with separate frontend and backend. The second part will be about Docker and how can we use it to develop the application and deploy it in production.
+
+**For first session working knowledge of JavaScript is a must. Please read up [https://developer.mozilla.org/en-US/docs/Web/JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript)**
+
+**For second session please install Docker for your operating system from for [here](https://www.docker.com/)**
+
+
+### Building JavaScript Application
+In this session we will learn how a modern web application is developed and deployed. The application will have a backend which will serve data via APIs and frontend will consume the data and present it using UI components. After developing the application we will deploy the application on [Heroku](https://www.heroku.com/) and [Firebase](https://firebase.google.com/) which allows a developer to deploy application without any prior knowledge of DevOps.
```

Comment:
SGTM

I think we should have name email and contact number fields in the form where name and email are mandatory ?